### PR TITLE
Add support for vulcanize v1.0

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,29 +17,21 @@ module.exports = function (opts) {
 			cb(new gutil.PluginError('gulp-vulcanize', 'Streaming not supported'));
 			return;
 		}
+	
+		vulcanize.setOptions(opts, function () {});
```
Comment (sindresorhus) [inline index.js]: the callback function is moot

File: index.js [hunk 2]
```diff
@@ -21,29 +17,21 @@ module.exports = function (opts) {
...
-		opts.outputHandler = function(filename, data, finished) {
-			this.push(new gutil.File({
-				cwd: file.cwd,
-				base: path.dirname(filename),
-				path: filename,
-				contents: new Buffer(data)
-			}));
-
-			if (finished) {
+		vulcanize.process(file.path, function(err, inlinedHtml) {
```
Comment (sindresorhus) [inline index.js]: `vulcanize.process(file.path, function (err, inlinedHtml) {`

File: index.js [hunk 3]
```diff
@@ -21,29 +17,21 @@ module.exports = function (opts) {
...
+			if (err) {
+				cb(new gutil.PluginError('gulp-vulcanize', err, {fileName: file.path}));
```
Comment (sindresorhus) [inline index.js]: early return instead of if/else

**Comment by addyosmani** [comment]
Thanks for working on this! Saved me some time re-doing my PR today :)

**Pull request closed without merging**